src/pages/workspace/WorkspacesListPage.js

```diff
@@ -289,49 +289,51 @@ function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount, r
 
     if (_.isEmpty(workspaces)) {
         return (
-            <IllustratedHeaderPageLayout
-                backgroundColor={theme.PAGE_THEMES[SCREENS.SETTINGS.WORKSPACES].backgroundColor}
-                illustration={LottieAnimations.WorkspacePlanet}
-                onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS.ROOT)}
-                title={translate('common.workspaces')}
-                style={!isSmallScreenWidth && styles.alignItemsCenter}
-                shouldShowBackButton={isSmallScreenWidth}
-                footer={
-                    isSmallScreenWidth && (
+            <ScreenWrapper shouldShowOfflineIndicatorInWideScreen>
```

Comment:
Sorry for the mislead here. Turns out `IllustratedHeaderPageLayout` internally uses `ScreenWrapper`.  All we actually need to do here is to pass `shouldShowOfflineIndicatorInWideScreen` to `IllustratedHeaderPageLayout`.

(Fixing https://github.com/Expensify/App/pull/33280#issuecomment-1885796157)